scripts/linters/python_linter.py

```diff
@@ -117,6 +117,41 @@ def _check_import_order(self):
                 summary_messages.append(summary_message)
         return summary_messages
 
+    def _check_non_test_files(self):
+        """This function is used to check that function
+           with test_only in their names are in test files.
+        """
+        if self.verbose_mode_enabled:
+            python_utils.PRINT('Starting function defintion checks')
+            python_utils.PRINT('----------------------------------------')
+        summary_messages = []
+        files_to_check = self.py_filepaths
+        failed = False
+        for filepath in files_to_check:
+            if filepath.find('_test.py') == -1:
+                with python_utils.open_file(filepath, 'r') as fp:
```

Comment:
Use `FILE_CACHE` instead of `open_file`, see how checks are implemented in `js_ts_linter` file.

Reply:
done.@PTAL